benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -167,6 +168,39 @@ def assert_created(returned_datasets):
         d2.addCallback(assert_created)
         s.stop()
 
+    @capture_logging(None)
+    def test_cleanup_deletes_dataset(self, _logger):
+        """
+        ``write_request_load_scenario`` deletes the dataset created by
+        the setup when the scenario is stopped.
+        """
```

Comment:
Yes, I think you're right Pilar. Descriptive names are good, but consistency is also good.  If the other tests use the same abbreviated names, then they should all be changed, as a separate issue, since the read_request_tests probably use the same scheme.